conan_v2.rst

```diff
@@ -20,11 +22,11 @@ If you want to migrate to 2.0, there are several things you will need to change:
     the more relevant changes.
   - **General changes** not related to the recipes nor the Conan commands, "build profiles", lowercase references... etc.
 
-If you are looking for precompiled binaries, there is a very short list in a separate remote which can be added (please, check the `Conan 2.0 documentation <https://docs.conan.io/en/2.0/index.html>`_ for more information)
 
-    .. code-block:: bash
+.. notice::
 
-      $ conan remote add conanv2 https://conanv2beta.jfrog.io/artifactory/api/conan/conan --index 0
+    If you are looking for precompiled binaries, please, check the
+    `Conan 2.0 documentation <https://docs.conan.io/2/index.html>`_ for more information.
```

Comment:
Maybe we can replace this with a note explaining that we already have lots of recipes prepared for v2, some of them with generated binaries, in conancenter and pointing to the list of packages that already have binaries available ?

Reply:
That sounds like a much better idea :) 

EDIT: after looking into it, I am not sure we even have that list

https://conan.io/cci-v2.html
https://github.com/conan-io/conan-center-index/discussions/16196

> List last updated 2023-03-03 14:17 UTC

I think that's less helpful